zero_to_chat_with_your_data/zero_to_chat_with_your_data.md

```diff
@@ -892,10 +917,83 @@ The Streamlit in Snowflake interface is divided into three panes:
 
 By default, only the Streamlit editor and preview panes are displayed. To change the display, use the show/hide buttons in the lower-left corner of the Streamlit in Snowflake editor.
 
-> 🚧 UNDER CONSTRUCTION: Provide a screenshot of the default Streamlit application
+![streamlit](assets/streamlit_2.png)
 
 Add snowpark-ml-python package from the packages dropdown in the code editor section
 
+### Cortex AI
+
+Snowflake offers two broad categories of powerful, intelligent features based on Artificial Intelligence (AI) and Machine Learning (ML). These features can help you do more with your data in less time than ever before.
+
+- Snowflake Cortex is a suite of AI features that use large language models (LLMs) to understand unstructured data, answer freeform questions, and provide intelligent assistance. This suite of Snowflake AI Features comprises:
+
+  - Snowflake Cortex LLM Functions
```

Comment:
Can you please add the LLM functions with a brief description from https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions:

- [COMPLETE](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#label-cortex-llm-complete): Given a prompt, returns a response that completes the prompt. This function accepts either a single prompt or a conversation with multiple prompts and responses.
- [EMBED_TEXT_768](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#label-cortex-llm-embed-text): Given a piece of text, returns a [vector embedding](https://docs.snowflake.com/en/user-guide/snowflake-cortex/vector-embeddings) of 768 dimensions that represents that text.
- [EMBED_TEXT_1024](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#label-cortex-llm-embed-text-1024): Given a piece of text, returns a [vector embedding](https://docs.snowflake.com/en/user-guide/snowflake-cortex/vector-embeddings) of 1024 dimensions that represents that text.
- [EXTRACT_ANSWER](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#label-cortex-llm-extract-answer): Given a question and unstructured data, returns the answer to the question if it can be found in the data.
- [SENTIMENT](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#label-cortex-llm-sentiment): Returns a sentiment score, from -1 to 1, representing the detected positive or negative sentiment of the given text.
- [SUMMARIZE](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#label-cortex-llm-summarize): Returns a summary of the given text.
- [TRANSLATE](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#label-cortex-llm-translate): Translates given text from any supported language to any other.

Reply:
Added